testing/frr/APKBUILD

```diff
@@ -0,0 +1,73 @@
+# Contributor: Arthur Jones <arthur.jones@riverbed.com>
+pkgname=frr
+pkgver=5.0
+pkgrel=0
+pkgdesc="Free Range Routing is a fork of quagga"
+url="https://frrouting.org/"
+arch="x86_64"
+license="GPL-2.0"
+depends="json-c c-ares ipsec-tools iproute2 python py-ipaddr bash"
+makedepends="ncurses-dev net-snmp-dev gawk texinfo perl
```

Comment:
The makedepends needs a cleanup. Just to mention some of the redundant dependencie (already part of the build enviroment or unneeded): sudo tar perl patch git curl

Reply:
Sure...